docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
 # FCA: Configurations
 
-TODO: add intro
+This document gives the various configurations available for Firebolt Certification App (FCA). It provides insights into setting up and utiizing different environments, parameters, and app types to facilitate effective testing and certification of applications using firebolt technology. 
+
 
 ## Table of Contents
 
-- [Supported Targets](#supported-targets)
+- [Running with MFOS](#running-with-mfos)
 - [Connect to Mock Firebolt](#connect-to-mock-firebolt)
 - [Supported URL Parameters](#supported-url-parameters)
 - [App Types](#app-types)
 
-## Supported targets
-
-EXISTING CONTENTS(needs revision):
-> - MFOS - `https://github.com/rdkcentral/mock-firebolt`
+## Running with MFOS
 
 ### Background
 
-TODO: add details
+Mock Firebolt (MFOS) is a simulated environment that FCA can interact with.
 
 ### Setup
 
-TODO: add details
+- *MFOS*: Accessible at [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
```

Comment:
Can make this a full sentence, Like:
The open source GitHub project is accessible at...